src/main/java/appeng/core/AELog.java

```diff
@@ -108,4 +108,14 @@ public static void debug( String format, Object... data )
 			log( Level.DEBUG, format, data );
 		}
 	}
+
+	public static void debugError( final Throwable e )
+	{
+		if( AEConfig.instance.isFeatureEnabled( AEFeature.DebugLogging ) )
+		{
+			severe( "DebugError: " + e.getClass().getName() + " : " + e.getMessage() );
+			e.printStackTrace();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1148)